backend/src/test/java/heartbeat/service/source/github/GithubServiceTest.java

```diff
@@ -12,10 +12,7 @@
 import heartbeat.client.dto.codebase.github.PullRequestInfo;
 import heartbeat.client.dto.pipeline.buildkite.DeployInfo;
 import heartbeat.client.dto.pipeline.buildkite.DeployTimes;
-import heartbeat.exception.InternalServerErrorException;
-import heartbeat.exception.NotFoundException;
-import heartbeat.exception.PermissionDenyException;
-import heartbeat.exception.UnauthorizedException;
+import heartbeat.exception.*;
```

Comment:
@weiraneve should not use `*` in any project, at least, our project, revert it

Reply:
done